docs/runtimes/load-from-hub.md

```diff
@@ -1,56 +1,70 @@
+
+<br>
+
+<img src="../_static/images/marketplace-ui.png" width="800">
+
 <br>
-![Hub](../_static/images/marketplace-ui.png)
+
 **In this section**
 
 - [Functions](#functions)
 - [Model monitoring modules](#model-monitoring-modules)
 - [Custom hub](#custom-hub)
 
+```{caution} 
+**About custom hubs**: If you don't specify a hub name at all, the algorithm searches for the function in all the hubs, starting with the most recently defined (custom) hub and going backwards in time. The MlRun hub is the last in the search. If you have multiple hubs, best practice is to specify the hub name when importing from any hub.
+```
+
 ## Functions
 
 There are functions for ETL, data preparation, training (ML & Deep learning), serving, alerts and notifications and more.
-Each function has a docstring that explains how to use it. The functions are categorized and their associated versions are listed, so you can easily find a suitable function/module for your needs.
-
-### Prerequisiteß
-
-- {ref}`Create your project <create-projects>`
+Each function has a docstring that explains how to use it. The functions are categorized and their associated versions are listed, so you can easily find a suitable function for your needs.
 
 ### Load a function from the MLRun hub
 
-This section demonstrates how to import a function from the hub into your project, and provides some basic instructions on how to run the function.
+There are two ways to import a function from the hub:
+- {py:meth}`~mlrun.projects.MlrunProject.set_function`: adds or update a function object to your project
+- {py:meth}`~mlrun.import_function`: creates a function object that you can use as relevant, for example running it as a job
 
-Run `project.set_function` to add or update a function object to your project.
-See all the parameters in {py:meth}`~mlrun.projects.MlrunProject.set_function`.
+#### Use `set_function`
 
-The `describe` function analyzes a csv or parquet file for data analysis.
-To load the `describe` function from the MLRun hub:
+This example runs the describe function. This function analyzes a dataset (in this case it's a csv file) and generates HTML files (e.g. correlation, histogram) and saves them under the artifact path.
 
 ```python
+# Load the `describe` function from the MLRun hub:
 project.set_function("hub://describe", "describe")
-```
 
-To load the same function from your [custom hub](#custom-hub):
-```python
+# Or load the same function from your [custom hub](#custom-hub):
+
 project.set_function("hub://<hub-name>/describe", "describe")
-```
-```{caution} 
-If you don't specify a hub name at all, the algorithm searches for the function in all the hubs, 
-giving preference to newly defined hubs. Therefore, if you 
-have multiple hubs, best practice is to explicitly mention the hub name.
+
+# Create a function object named, for example, `my_describe`:
+my_describe = project.func("describe")
```

Comment:
to be honest I have no idea what is this .func() method under the project SDK. You should change it to `project.get_function("describe")`